src/hotspot/share/cds/archiveHeapLoader.cpp

```diff
@@ -378,11 +378,10 @@ void ArchiveHeapLoader::finish_initialization() {
     // The heap roots are stored in one or more segments that are laid out consecutively.
     // The byte size of each segment (except for the last one) is max_size.
```

Comment:
Comment here mentions `max_size` but that doesn't exist any more.

Reply:
Thanks! I tightened up the comment a bit.